CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing guidelines
+
+## Preparing pull requests
+
+1. Follow the [formatting/style settings](.vscode/settings.json), run (SHIFT+ALT+F) and also look how everything else is formatted.
```

Comment:
Maybe mention that SHIFT+ALT+F is an example of how to do it with vscode and the keyboard shortcuts may differ based on editor used.

Reply:
Agree, fixed